assets/tests/abac_tests.yaml

```diff
@@ -1092,6 +1092,184 @@ tests:
             context:
               "x": 20
             expectation:
+  - name: simple_ttu_with_and_without_condition
+    stages:
+      - model: |
+          model
+            schema 1.1
+          type user
+          type folder
+            relations
+              define viewer: [user, user with xcond]
+          type document
+            relations
+              define parent: [folder]
+              define viewer: viewer from parent
+          condition xcond(x: int) {
+            x == 10
+          }
+        tuples:
+          - user: folder:a
+            relation: parent
+            object: document:1
+          - user: user:without-condition
+            relation: viewer
+            object: folder:a
+          - user: user:with-condition
+            relation: viewer
+            object: folder:a
+            condition:
+              name: xcond
+        checkAssertions:
+          - tuple:
+              user: user:with-condition
+              relation: viewer
+              object: document:1
+            errorCode: 2000
+          - tuple:
+              user: user:with-condition
+              relation: viewer
+              object: document:1
+            context:
+              "x": 10
+            expectation: true
+          - tuple:
+              user: user:without-condition
+              relation: viewer
+              object: document:1
+            expectation: true
+          - tuple:
```

Comment:
how is this different than the tuple above?

Reply:
Removed.